server/fakedb/Tag.csv

```diff
@@ -0,0 +1,24 @@
+﻿name,eventId
```

Comment:
In general, we can add a functionality to support comment lines, so we can skip over lines that starts with "#" for example.

Reply:
And leave a comment saying each line is {Topic} and {event ID}, and since we support multiple tags per event, there may be multiple lines per event.